src/pages/analytics/Analytics.js

```diff
@@ -30,26 +31,39 @@ const startAnalyticsTablesGenerationMutation = {
     params: params => params,
 }
 
+const existingTasksQuery = {
+    tasks: {
+        resource: 'system/tasks/ANALYTICS_TABLE',
+    },
+}
+
 const Analytics = ({ sectionKey }) => {
     const { checkboxes, toggleCheckbox } = useCheckboxes()
     const [lastYears, setLastYears] = useState(DEFAULT_LAST_YEARS)
+    const poll = useAnalyticsPoll()
+    useDataQuery(existingTasksQuery, {
+        onComplete: data => {
+            const taskId = getActiveTaskIdFromSummary(data.tasks)
+
+            if (taskId) {
+                poll.start({ taskId })
+            }
+        },
+    })
     const [
         startAnalyticsTablesGeneration,
-        { loading, error, data },
+        { loading, error },
     ] = useDataMutation(startAnalyticsTablesGenerationMutation, {
         onComplete: data => {
-            const { id: jobId } = data.response
-            poll.start({ jobId })
+            const { id: taskId } = data.response
+            poll.start({ taskId })
         },
     })
-    const poll = useAnalyticsPoll()
     const notifications = poll.data ? getUpdatedNotifications(poll.data) : []
-    const isPolling =
-        data && (!poll.data || !notifications.some(n => n.completed))
 
     const handleStartAnalyticsTablesGeneration = () => {
         const params = {}
-        for (const [key, checked] of Object.entries(checkboxes)) {
+        for (const [key, { checked }] of Object.entries(checkboxes)) {
```

Comment:
This is where the cause of the bug regarding the analytics parameter values always being `true`